apps/mdn/mdn-aws/k8s/percona-toolkit-ptkill.yaml.j2

```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       containers:
       - name: percona-toolkit-ptkill
-        image: {{ env['PERCONA_TOOLKIT_IMAGE'] }}
+        image: {{ PERCONA_TOOLKIT_IMAGE }}
         env:
           - name: PT_CMD
```

Comment:
This whole section for `PT_CMD` (lines 22-26) should be deleted

Reply:
Good catch :)